### ISSUE TITLE
ValidationElements-Iterator seems broken when using ExternalDataConnector-Interfaces

### ISSUE THREAD

**Issue by MenschMitHut** [AutomationML/AMLEngine2.1#4]
Hi,

it is not possible to iterate through the validationElements if an InternalElement has a ExternalDataConnector-Interface.
With all other interfaces the iteration is possible.

```
var caexDocument = CAEXDocument.New_CAEXDocument();            
// add an InstanceHierarchy to the ROOT CAEXFile element
var myIH = caexDocument.CAEXFile.New_InstanceHierarchy("myIH");

// add an InternalElement
var ie = myIH.InternalElement.Append("ie");

// With these interfaces the exception will be thrown
ie.New_ExternalInterface("Test").MakeCOLLADAInterface();
//ie.New_ExternalInterface("Test").MakeExternalDataConnector();
//ie.New_ExternalInterface("Test").MakePLCopenXMLInterface();

ValidatorService service = ValidatorService.Register();
var validationElements = service.ValidateAll(caexDocument);
//Exception will be thrown in the next line
int count = validationElements.Count();
ValidatorService.UnRegister();
```

```
System.NullReferenceException
  HResult=0x80004003
  Message=Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
  Source=AML.Engine.Services
  StackTrace:
   at Aml.Engine.Services.ValidatorService.<>c.<GetExternalDataReferenceValidationIssues>b__24_0(ExternalInterfaceType ext)
   at System.Linq.Enumerable.<>c__DisplayClass6_0`1.<CombinePredicates>b__0(TSource x)
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Lookup`2.Create[TSource](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.GroupedEnumerable`3.GetEnumerator()
   at Aml.Engine.Services.ValidatorService.<GetExternalDataReferenceValidationIssues>d__24.MoveNext()
   at Aml.Engine.Services.ValidatorService.<GetAllValidationIssues>d__18.MoveNext()
   at System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   at TestApp.Program.Main(String[] args) in ...\visual studio 2017\Projects\TestApp\Program.cs:line 31
```

**Comment by jprinz** [comment]
Fixed in v1.0.17

**Comment by MenschMitHut** [comment]
Thanks,

i can confirm its working
